diagnosis_api/tests/test_disease_details_routes.py

```diff
@@ -5,36 +5,44 @@
 
 
 class TestApiRoutes(unittest.TestCase):
-
     def setUp(self):
         self.app_client = app.test_client()
-    
+
     def test_index(self):
 
         index = self.app_client.get("/index", content_type="application/json")
         self.assertEqual(index.status_code, 200)
         response = json.loads(index.data)
-        self.assertEqual(response['message'], 'Welcome to disease diagnosis API with pandas')
+        self.assertEqual(
```

Comment:
You could also go ahead to test the status code from the response as well as the data from the response if it exists or  not

Reply:
I have been trying as much as possible to avoid the data returned because i wanted this to be a template such that if i load any animal diseases file, it can extract the features i need. Would that be reasonable?

Reply:
Yes, but you also want your tests to mimic the way the user would interact with the system and see that the results are acting in the expected behavior.

Reply:
Did you mean adding assertions for testing if the object IsNotNone?